SpeechResponder/ScriptResolver.cs

```diff
@@ -917,7 +917,7 @@ private BuiltinStore buildStore(Dictionary<string, Cottle.Value> vars)
                         result.bodyshortname = body.shortname;
                     }
                 }
-                return new ReflectionValue(result);
+                return result == null ? new ReflectionValue(new object()) : new ReflectionValue(result);
```

Comment:
How about 
```cs
return new ReflectionValue(result ?? new object());
```
Feels more idiomatic.

Reply:
Done, though I'm not sure you noticed the other methods using the same construction. ;-)